Makefile

```diff
@@ -35,10 +35,10 @@ endpoints: check-env ## Create alert endpoints and integrations
 	@${PIPENVCMD} ansible-playbook -i localhost, setup_alert_endpoints.yml --diff ${EXTRA}
 
 routes: check-env ## Create alert routes
-	@${PIPENVCMD} ansible-playbook -vvv -i localhost, setup_alert_routes.yml --diff ${EXTRA}
```

Comment:
Just an heads up in case you didn't mean to change this

Reply:
Yes, I did. Too much logging could leak security tokens to the screen. It's better to keep it at a minimum.